src/pam-configs/mfa-google-authenticator

```diff
@@ -0,0 +1,13 @@
+Name: Google Authenticator PAM module (2FA/MFA)
+Default: no
+Priority: 9999
+
+Auth-Type: Primary
+Auth:
+    required      pam_google_authenticator.so nullok forward_pass
```

Comment:
This is equal to:

```
[default=bad success=ok new_authtok_reqd=ok ignore=ignore] pam_google_authenticator.so nullok forward_pass
```

But I would prefer:

```
[default=ignore success=ok auth_err=die] pam_google_authenticator.so nullok forward_pass
```

The module returns only `PAM_SUCCESS`, `PAM_IGNORE`, and `PAM_AUTH_ERR`. The main difference is that we will not call `unix` or other modules if the MFA code is wrong.